CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#1475](https://github.com/openmls/openmls/pull/1475): Fully process GroupContextExtension proposals
 - [#1477](https://github.com/openmls/openmls/pull/1477): Allow setting leaf node extensions and capabilities of the group creator when creating an MlsGroup(Config)
 - [#1478](https://github.com/openmls/openmls/pull/1478): Remove explicit functions to set `RequiredCapabilitiesExtension` and `ExternalSendersExtension` when building an MlsGroup(Config) in favor of the more general function to set group context extensions
+- [#1479](https://github.com/openmls/openmls/pull/1479): Allow the use of extensions with `ExtensionType::Unknown` in group context and leaf nodes
```

Comment:
Is there a reason to restrict to group context and leaf nodes?

Reply:
Now that I think about it, I could include KeyPackage extensions. It probably also works for GroupInfo extensions, but we don't have a way to send specific GroupInfo extensions.

Reply:
I've changed the changelog accordingly and added a test, as well as documentation in the book w.r.t. unknown KP extensions. I suspect that GroupInfo extensions also work, but we'd have to expose them at the API level first.